io_scene_dae/export_dae.py

```diff
@@ -294,103 +294,104 @@ def export_material(self, material, double_sided_hint=True):
             if ts.use_map_normal and normal_tex is None:
                 normal_tex = sampler_sid
 
-        self.writel(S_FX, 3, '<technique sid="common">')
+        self.writel(S_FX, 3, "<technique sid=\"common\">")
         shtype = "blinn"
-        self.writel(S_FX, 4, '<' + shtype + '>')
+        self.writel(S_FX, 4, "<{}>".format(shtype))
         # Ambient? from where?
 
-        self.writel(S_FX, 5, '<emission>')
+        self.writel(S_FX, 5, "<emission>")
         if emission_tex is not None:
-            self.writel(S_FX, 6, '<texture texture="' + emission_tex +
-                        '" texcoord="CHANNEL1"/>')
+            self.writel(
+                S_FX, 6, "<texture texture=\"{}\" texcoord=\"CHANNEL1\"/>"
+                .format(emission_tex))
         else:
-            self.writel(S_FX, 6, '<color>' +
-                        numarr_alpha(material.diffuse_color, material.emit) +
-                        ' </color>')  # not totally right but good enough
-        self.writel(S_FX, 5, '</emission>')
-
-        self.writel(S_FX, 5, '<ambient>')
-        self.writel(S_FX, 6, '<color>' +
-                    numarr_alpha(self.scene.world.ambient_color,
-                                 material.ambient) + ' </color>')
-        self.writel(S_FX, 5, '</ambient>')
-
-        self.writel(S_FX, 5, '<diffuse>')
+            # not totally right but good enough
+            self.writel(S_FX, 6, "<color>{} </color>".format(
```

Comment:
I wonder if the extra whitespace after {} is necessary (this was not introduced by @set-killer).